demo/src/demo-score.ts

```diff
@@ -1,93 +1,160 @@
+    await createDid(collectorIdentity)
+
+  console.log('✅  Entities created!')
+
+  console.log(`\n❄️  Registry Creation `)
+
+  const registryTitle = `Registry v3.${randomUUID().substring(0, 4)}`
+  const registryDetails: Cord.IContents = {
+    title: registryTitle,
+    description: 'Registry for for scoring',
   }
-  console.dir(journalContent, { depth: null, colors: true })
 
-  let newJournalEntry = Cord.Score.fromJournalProperties(
-    journalContent,
-    sellerIdentity
-  )
+  const registryType: Cord.IRegistryType = {
+    details: registryDetails,
+    creator: issuerDid.uri,
+  }
 
-  let journalCreationExtrinsic = await Cord.Score.entries(newJournalEntry)
-  console.log(`\n❄️  Transformed Journal Entry `)
-  console.dir(newJournalEntry, { depth: null, colors: true })
+  const txRegistry: Cord.IRegistry =
+    Cord.Registry.fromRegistryProperties(registryType)
 
+  let registry
   try {
-    await Cord.Chain.signAndSubmitTx(
-      journalCreationExtrinsic,
-      transactionAuthor,
-      {
-        resolveOn: Cord.Chain.IS_IN_BLOCK,
-        rejectOn: Cord.Chain.IS_ERROR,
-      }
+    await Cord.Registry.verifyStored(txRegistry)
+    console.log('Registry already stored. Skipping creation')
+  } catch {
+    console.log('Regisrty not present. Creating it now...')
+    // Authorize the tx.
+    const tx = api.tx.registry.create(txRegistry.details, null)
+    const extrinsic = await Cord.Did.authorizeTx(
+      issuerDid.uri,
+      tx,
+      async ({ data }) => ({
+        signature: issuerKeys.assertionMethod.sign(data),
+        keyType: issuerKeys.assertionMethod.type,
+      }),
+      authorIdentity.address
     )
-    console.log('✅ Journal Entry added!')
-  } catch (e: any) {
-    console.log(e.errorCode, '-', e.message)
+    console.log('\n', txRegistry)
+    // Write to chain then return the Schema.
+    await Cord.Chain.signAndSubmitTx(extrinsic, authorIdentity)
+    registry = txRegistry
   }
+  console.log('\n✅ Registry created!')
 
-  console.log(`\n❄️  Query Chain Scores `)
-  const chainScore = await Cord.Score.query(
-    journalContent.entity,
-    journalContent.scoreType
+  // Step 4: Add Delelegate One as Registry Admin
+  console.log(`\n❄️  Registry Admin Delegate Authorization `)
+  const registryAuthority = await addRegistryAdminDelegate(
+    authorIdentity,
+    issuerDid.uri,
+    registry['identifier'],
+    delegateOneDid.uri,
+    async ({ data }) => ({
+      signature: issuerKeys.capabilityDelegation.sign(data),
+      keyType: issuerKeys.capabilityDelegation.type,
+    })
   )
-  console.dir(chainScore, { depth: null, colors: true })
+  console.log(`\n✅ Registry Authorization - ${registryAuthority} - created!`)
 
-  const chainAvgScore = await Cord.Score.queryAverage(
-    journalContent.entity,
-    journalContent.scoreType
+  console.log(`\n❄️  Journal Entry `)
+  let journalContent: IJournalContent = {
+    entity: sellerDid.uri.replace('did:cord:', ''),
```

Comment:
entity Identifier is not DID atm. It's an identifier generated from a Govt ID without any keys attached to it. Please refer the specs.